website/events/admin/forms.py

```diff
@@ -49,3 +50,31 @@ class Meta:
         widgets = {
             "type": forms.Select,
         }
+
+
+class EventAdminForm(forms.ModelForm):
+    def clean(self):
+        super().clean()
+        self.instance.clean_changes(self.changed_data)
+
+    def clean_organisers(self):
+        if self.request.user.is_superuser or self.request.user.has_perm(
+            "events.override_organiser"
+        ):
+            return self.cleaned_data.get("organisers")
+
+        if (
+            self.request.member
+            and self.cleaned_data.get("organisers")
+            and not (
+                self.request.member.get_member_groups()
+                .filter(pk__in=self.cleaned_data.get("organisers").values_list("pk"))
+                .exists()
+            )
```

Comment:
It would be neater to split the checks for 'only one' and 'cannot remove yourself', that way this 'if' is easier to read and there can be two error messages shown instead of one that says two things and then leaves you to figure our what exactly (of the two things) you did wrong.